examples/two-step-transfer/index.html

```diff
@@ -5,12 +5,42 @@
         <script src="../../node_modules/@concordium/web-sdk/lib/concordium.min.js"></script>
         <meta charset="utf-8" />
         <script>
-            function concordiumReady() {
+            // TODO Use the web-sdk when this function has been added there.
+            function detectConcordiumProvider(timeout = 5000) {
+                return new Promise((resolve, reject) => {
+                    let alreadyHandled = false;
+
+                    function handleConcordium() {
+                        if (alreadyHandled) {
+                            return;
+                        }
+                        alreadyHandled = true;
+
+                        if (window.concordium) {
+                            resolve(window.concordium);
+                        }
+                        reject();
+                    }
+
+                    if (window.concordium) {
+                        handleConcordium();
+                    } else {
+                        window.addEventListener('concordium#initialized', handleConcordium, { once: true });
+                        setTimeout(() => {
+                            handleConcordium();
+                        }, timeout);
+                    }
+                });
+            }
```

Comment:
I guess if you end up changing the corresponding function in the piggy bank example, you should change this one too.